RSFormView/Classes/Views/TextField/TextFieldView.swift

```diff
@@ -247,14 +249,26 @@ extension TextFieldView: UIPickerViewDelegate, UIPickerViewDataSource {
 }
 
 extension TextFieldView: UITextFieldDelegate {
-  public func textField(_ textField: UITextField,
-                 shouldChangeCharactersIn range: NSRange,
-                 replacementString string: String) -> Bool {
-    guard let data = fieldData,
-      data.fieldType != .date, data.fieldType != .picker else {
+  public func textField(
+    _ textField: UITextField,
+    shouldChangeCharactersIn range: NSRange,
+    replacementString string: String
+  ) -> Bool {
+    guard
+      let data = fieldData,
+      data.fieldType != .date,
+      data.fieldType != .picker
+    else {
         return false
     }
     
+    if lastEntry == string && (string.count > 1 || string == " ") {
+      //when using swipe keyboard on iOS 13 we get updates sending the same entered word or spaces a bunch of times
+      return false
+    }
```

Comment:
wouldn't this affect copy-pasting in the field?

Reply:
nope, it works well

Reply:
you made me nervous for a second though